back/src/adapters/primary/routers/apiKeyAuthRouter/createApiKeyAuthRouter.v2.ts

```diff
@@ -249,3 +253,16 @@ const getFirstLocationIdOrThrow = async (
     );
   });
 };
+
+const addAcquisitionParams = <T>(
+  obj: T,
+  apiConsumer: ApiConsumer | undefined,
+): T & WithAcquisition => ({
+  ...obj,
+  ...(apiConsumer
+    ? {
+        acquisitionCampaign: "api-consumer",
+        acquisitionKeyword: apiConsumer.id,
```

Comment:
peut-être plus simple de lire la donnée:

```suggestion
        acquisitionKeyword: `${apiConsumer.id} - {apiConsumer.name}`,
```

Reply:
c'est fixé